week2/js/index.js

```diff
@@ -0,0 +1,245 @@
+//--로컬 스토리지--
+
+//여기서 members는 data js에 있는 웨비들 다 묶은 변수명
+import { members } from './data.js';
+
+const data = JSON.stringify(members);
+if (!localStorage.getItem('webby')) {
+  window.localStorage.setItem('webby', data);
+}
+
+const webbyData = window.localStorage.getItem('webby');
+const dataList = JSON.parse(webbyData); //여러 변수를 거쳐, 웨비들의 데이터 변수명은 dataList가 됨!! 기억하고 있자
```

Comment:
이렇게 여러 변수를 거쳐서 dataList를 뽑아내는것도 좋은 방식입니다!

제가 한 방식을 보여드리자면
```suggestion
if (!localStorage.getItem("membersData")) {
  localStorage.setItem("membersData", JSON.stringify(members));
}
let membersData = JSON.parse(localStorage.getItem("membersData"));
```
이렇게 했는데 지유님의 코드와 똑같은 행위를 단지 짧게 했을 뿐이긴 한데 어떤 방식으로 하든 상관은 없을 것 같아요.
오히려 지유님의 코드가 각각의 변수에 어떤 값이 담길지 유추하기는 더 쉬울 수 있을 것 같네요.
대신 변수를 많이 생성한 만큼 메모리가 사용된다는 점 알아두면 좋을 것 같습니다!